cloudwatch_sns_to_slack/handler.py

```diff
@@ -1,23 +1,28 @@
 import json
 import logging
 from os import getenv
+from uuid import uuid4
 
+import boto3
 import requests
 
 from cloudwatch_sns_to_slack.constants import (
     BASE_CLOUDWATCH_ALARM_LINK,
     DEV_SNS_ARN,
     GENERIC_ERROR_MESSAGE,
     HEADERS,
-    SERVICE_NAME
+    SERVICE_NAME,
+    S3_BUCKET_NAME
 )
 
 
 logger = logging.getLogger(SERVICE_NAME)
 logger.setLevel(logging.INFO)
 
 SLACK_WEBHOOK_URL = getenv('SLACK_WEBHOOK_URL')
-PLATFORM_ALERTS_CHANNEL = '#platform-alerts'
+SRE_ALERTS_CHANNEL = '#sre-alerts'
```

Comment:
Does this act as the default channel in case of messages coming to this lambda without a channel specified? If so, nitpicking, but maybe it could make sense to have this variable called `DEFAULT_ALERTS_CHANNEL`? Just to prevent the question that popped into my head, which was "how come we only need to hard-code the SRE channel here? 🤔 "

Reply:
Also makes following changes in the actual name of the default channel easier to refactor in the future

Reply:
> Does this act as the default channel in case of messages coming to this lambda without a channel specified

yup! and good call :)